Tutorial_1a_Convolving_Images.ipynb

```diff
@@ -0,0 +1,650 @@
+{
```

Comment:
<p>Would you mind adding a 2D examplle with spectral-cube, too? As it's a quick couple of lines:</p>

<pre class="ql-syntax" spellcheck="false">from spectral_cube import Projection
from astropy.io import fits
from radio_beam import Beam
import astropy.units as u

img = Projection.from_hdu(fits.open('file.fits'))
targ_beam = Beam(33*u.arcsec)
conv_img = img.convolve_to(targ_beam)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>